helm_deploy/court-case-matcher/templates/ingress.yaml

```
@@ -40,7 +39,9 @@ spec:
         paths:
           - path: {{ $ingressPath }}
```

Missing `pathType: ImplementationSpecific`
```suggestion
          - path: {{ $ingressPath }}
            pathType: ImplementationSpecific
```